app/src/main/AndroidManifest.xml

```diff
@@ -49,7 +49,7 @@
             android:exported="true" />
 
         <activity
-            android:name=".ui.activity.SettingsActivity"
+            android:name=".ui.fragment.SettingsFragment"
```

Comment:
@dcetkovic what is purpose of having Fragment in AndroidManifest?

Reply:
I missed that. There is no purpose, we do not register fragment in manifest file, only its host activity. I did aut refactoring, so that is why name changed in manifest.

Reply:
Will remove it.